build/Dockerfile

```diff
@@ -145,6 +146,10 @@ RUN if [[ "${ENABLE_AIM}" == "true" ]]; then \
         python -m pip install --user "$(head bdist_name)[aim]"; \
     fi
 
+RUN if [[ "${ENABLE_GPTQ}" == "true" ]]; then \
+    python -m pip install --user "$(head bdist_name)[gptq]"; \
+    fi
+
```

Comment:
Although this is a nice flag to have, I don't think we need it as users won't be enabling this flag. Since this is similar to a dev dependency, it's something users would have to manually install themselves if they want to use it. Doesn't hurt to have it here but wanted to note how it's different from aim and fms-acceleration.